arches/app/templates/views/search.htm

```diff
@@ -1,32 +1,13 @@
-<!--
-ARCHES - a program developed to inventory and manage immovable cultural heritage.
-Copyright (C) 2013 J. Paul Getty Trust and World Monuments Fund
-
-This program is free software: you can redistribute it and/or modify
-it under the terms of the GNU Affero General Public License as
-published by the Free Software Foundation, either version 3 of the
-License, or (at your option) any later version.
-
-This program is distributed in the hope that it will be useful,
-but WITHOUT ANY WARRANTY; without even the implied warranty of
-MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-GNU Affero General Public License for more details.
-
-You should have received a copy of the GNU Affero General Public License
-along with this program. If not, see <http://www.gnu.org/licenses/>.
--->
 {% extends "base-manager.htm" %}
 {% load i18n %}
 {% load template_tags %}
 {% load static %}
 
 {% block title %} {{ block.super }} {% trans "Search" %} {% endblock title %}
 
-
 {% block main_content %}
-    <div class="content-panel flexrow" data-bind="click:function() { menuActive(false); }, visible: true", style="display:none;">
-
-        <div class="flex search-results-panel">
+    <article class="content-panel flexrow" data-bind="click:function() { menuActive(false); }, visible: true", style="display:none;">
```

Comment:
Considering the following from Mozilla should we use section instead of article:

`The <article> [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) element represents a self-contained composition in a document, page, application, or site, which is intended to be independently distributable or reusable (e.g., in syndication). Examples include: a forum post, a magazine or newspaper article, or a blog entry, a product card, a user-submitted comment, an interactive widget or gadget, or any other independent item of content.`

Also, should we be using the `aria-label` property on these section and article elements to further describe their meaning?

Reply:
There are sections under the article. So using the css classes to refer to the containers, "content-panel flexrow" becomes the article and there are 2 sections within, one for "flex search-results-panel" and one for "flex".

We've not added aria-label's to containers previously as there were no audit issues raised, but it does make sense to do so as it further explains the regions on the page, so I will add them accordingly. I'll resolve this when changes pushed etc.